toolkit/imageconfigs/full.json

```diff
@@ -17,17 +18,20 @@
             },
             "KernelOptions": {
                 "default": "kernel"
-            }
+            },
+            "EnableGrubMkconfig": true
```

Comment:
Typically this EnableGrubMkconfig field is true by default; is this needed due to the empty systemconfig struct you discovered? And  "applyConfiguration" is what addresses this?

Reply:
Yes, this change is required in addition to the change in attendedinstaller.go. attendedinstaller.go copies the configuration from the selected build configuration over the empty systemconfig (where the grubmkconfig is false from its empty state from go initialization).

Reply:
Validated, these are being set to `true` by default. However, @camelron I did notice an edge case we didn't consider in the validator. An ISO config looks the same as a RootFS config, so we are skipping the grub package check.